Public/Src/Engine/Dll/Engine.cs

```diff
@@ -1471,19 +1471,21 @@ private JournalState GetJournalStateWithVolumeMap(VolumeMap volumeMap, LoggingCo
             Contract.Requires(loggingContext != null);
 
             // Under some configurations, we access the change journal (i.e., actually scan the journal).
-            var journal = Configuration.Engine.ScanChangeJournal
-                ? JournalAccessorGetter.TryGetJournalAccessor(
-                    loggingContext,
+            if (Configuration.Engine.ScanChangeJournal)

```

Comment:
if (OperatingSystemHelper.IsUnixOS || !Configuration.Engine.ScanChangeJournal) and then "if (Configuration.Engine.ScanChangeJournal)" and the last "return JournalState.DisabledJournal;" can be removed. It is easier to read in this logic.

Reply:
I have a different opinion. The Unix exclusion can be thought of as a shortcut for Unix. Thus, the rest of the code is only applied to Win.

---
In reply to: [293577631](https://github.com/microsoft/BuildXL/pull/457#discussion_r293577631) [](ancestors = 293577631)